package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "compile": "tsc",
-    "prettier": "prettier --write 'src/**/*.ts'",
+    "prettier": "prettier --write src/**/*.ts",
     "lint": "eslint --fix --format=pretty --ext .ts src/",
-    "check": "run-s prettier lint"
+    "check": "run-s prettier lint",
+    "task reverse string": "nodemon src/Module1/Task1-1.ts"
```

Comment:
Could you please rename the task without spaces

Reply:
I suppose this is for convenience to use via terminal/console ?